tests/integration/scripts/test_validate_integration.py

```diff
@@ -25,34 +25,32 @@
 
 
 async def test_revalidate_checks_source(
-    _source_workflow, capsys, _setup_validate_cli
+    capsys, validate, workflow_source, cylc_install
 ):
     """Validation fails if revalidating with broken config.
     """
-    wf = _source_workflow()
-
-    setup = _setup_validate_cli({'against_source': True})
+    src_dir = workflow_source()
+    run_dir = cylc_install(src_dir)
```

Comment:
I think it might make more sense to return the workflow ID rather than the workflow run-dir from `cylc_install`.

That way `cylc_install` matches the return value of `flow` and the `id` is the thing you need to provide to Cylc commands e.g. `cylc play`.